lambda/index.js

```diff
@@ -0,0 +1,3 @@
+const redirect = require('./src/redirect');
+
+exports.handler = (event, context, callback) => callback(null, redirect(event["queryStringParameters"]));
```

Comment:
Initially I was skeptical that the callback is always called with a success and no error. But after reading the lambda source below it makes sense - even if something goes wrong, the lambda needs to 'succeed' so that we can pass a sensible response back to the end user 👍